### ISSUE TITLE
video tag doesn't work with cube theme

### ISSUE THREAD

**Issue by linclark** [bespokejs/bespoke-theme-cube#2]
I was working on a presentation with a video in it. When using the cube theme, neither a YouTube embed nor a video tag worked.

Here's the Jade:

```
        video(width="320", height="240", controls)
          source(src="videos/installing-node.mp4", type="video/mp4")
          |Your browser does not support the video tag.
```

And the generated HTML:

```
<video width="320" height="240" controls>
          <source src="videos/installing-node.mp4" type="video/mp4">Your browser does not support the video tag.
</video>
```

I switched to the terminal theme and the video tag worked.

**Comment by markdalgleish** [comment]
Are you seeing the video at all? I tried using a video tag and found that I could see the video but couldn't click on it.

I added the following CSS to fix it, can you let me know if it fixes the issue for you too? If it does, I'll update and release a fix.

``` css
.bespoke-theme-cube-slide-parent {
  pointer-events: none;
}

.bespoke-slide {
  pointer-events: auto;
}
```

**Comment by linclark** [comment]
Thanks for the quick response, that does work for me.

**Comment by markdalgleish** [comment]
Just fixed this and published v2.0.1 to npm
